README.rst

```diff
@@ -0,0 +1,113 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME
+PDFmorph is a Python software package designed to increase the insight
+researchers can get into spectra collected for atomic pair distribution 
+functions.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+PDFmorph can then be started from a terminal...
+[insert more instructions here]
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+
+DEVELOPMENT
+------------------------------------------------------------------------
+
+PDFmorph is an open-source software available in a git repository at
+https://github.com/diffpy/diffpy.pdfmorph.
+
+Feel free to fork the project and contribute! To install PDFmorph
+in a development mode where the source files are used directly
+rather than copied to a system directory, use ::
+
+     python setup.py develop --user
+
+
+CONTACTS
+------------------------------------------------------------------------
+
+For more information on PDFmorph, visit the project wep-page:
+
+# FIXME - make a page for the project
+[insert project page here, when it exists]
+
+or email Professor Simon Billinge at sb2896@columbia.edu
+
+
+LICENSE
```

Comment:
This should be pulled from the license file outright. Bifurcation of the license text would be bad.

Reply:
Actually I would just include a link to the license.

Reply:
the link to the license file on the github in lieu of all this? should we not edit the license text everywhere to best reflect the years in development?

Reply:
There is a license file here: https://github.com/diffpy/diffpy.pdfmorph/blob/master/LICENSE.txt
I would edit that file and provide links to it. If there are two things called license and they say different things, which one is the correct license?